client/src/index.js

```diff
@@ -27,12 +28,20 @@ const Routes = () => {
               Status
             </Link>
           </li>
+          <li>
+            <Link className="nav-link" to="/createOpprtunity">
+              Create Opprtunity
+            </Link>
+          </li>
         </ul>
 
         <div>
-          <Route path="/" exact component={Home} />
-          <Route path="/about/" component={About} />
-          <Route path="/status/" component={Status} />
+          <Switch>
+            <Route path="/" exact component={Home} />
+            <Route path="/about/" component={About} />
+            <Route path="/status/" component={Status} />
+            <Route Path="/createOpprtunity" component={CreateOppForm} />
```

Comment:
misspelled - also route should `create-opportunity` (make sure the link is updated too)